endpoints/openrtb2/sample-requests/invalid-whole/format-two-widths.json

```diff
@@ -1,6 +1,6 @@
 {
-  "message": "Invalid request: Request imp[0].banner.format[0] should define *either* {w, h} *or* {wmin, wratio, hratio}, but not both. If both are valid, send two \"format\" objects in the request.\n",
-  "requestPayload": {
+  "description": "Banner comes with no width nor wratio value",
```

Comment:
Description is incorrect. It comes with both specified, which is invalid.

Reply:
Copy-paste error. Corrected